lib/fs.ts

```diff
@@ -176,9 +184,13 @@ class FSPath {
 			pendingOptions: null,
 			throttleTime,
 			throttleTimer: null,
+			finish: (async () => {
```

Comment:
This seems like a slower way to write

`finish: this.safeWrite(dataFetcher(), options).then(() => this.finishUpdate()),`